rt_gene/src/rt_gene/extract_landmarks_method.py

```diff
@@ -34,29 +34,90 @@
 from rt_gene.msg import MSG_SubjectImagesList
 from rt_gene.cfg import ModelSizeConfig
 from rt_gene.subject_ros_bridge import SubjectListBridge
-from rt_gene.tracker import GenericTracker, TrackedElement
+from rt_gene.GenericTracker import TrackedElement
+from rt_gene.FaceEncodingTracker import FaceEncodingTracker
+from rt_gene.SequentialTracker import SequentialTracker
 
 import face_alignment
 from face_alignment.detection.sfd import FaceDetector
 
-import torch
+import dlib
+
 
 class TrackedSubject(TrackedElement):
-    def __init__(self, box, face, landmarks, marks):
+
+    def __init__(self, box, face, landmarks, marks, face_encoder):
         super(TrackedSubject, self).__init__()
         self.box = box
         self.face_color = face
         self.transformed_landmarks = landmarks
         self.marks = marks
+        self.face_encoder = face_encoder
+
+    def _align(self, desired_left_eye=(0.3, 0.3), desired_face_width=150, desired_face_height=150):
```

Comment:
I might be confused, but isn't there an aligning method already somewhere?

Reply:
> I might be confused, but isn't there an aligning method already somewhere?

Not that I know of but I could be entirely wrong! The `_align` method is now moved to `FaceEncodingTracker` anyway to not polute the main code repo...if there is a similar `align` method that I could reuse i'll just call that from the `FaceEncodingTracker` as an improvement?